src/data-workspace/inputs/inputs.module.css

```diff
@@ -61,6 +61,10 @@
     flex-grow: 1;
 }
 
+.select {
+    max-width: 150px;
```

Comment:
Hmm...we do want this. I mean I get that it's a bit distracting for the width to grow, but it does help show more of the selections?

Reply:
Yeah I'm a bit unsure what to do here. This was just an initial pass on the styling. Should probably consult @cooper-joe , but we definitely cannot show all options. And I do think it could be a bit distracting that it grows so much. Can probably be a bit wider though.

Another option would be to enable "wrapping" and have it grow vertically instead of horizontally. Also probably a better approach for printing the form, as we would need to include all options in print-format.